<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qt_testClass</class>
 <widget class="QMainWindow" name="qt_testClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qt_test</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qt_test.qrc">
    <normaloff>:/qt_test/GeneratedFiles/icon.bmp</normaloff>:/qt_test/GeneratedFiles/icon.bmp</iconset>
  </property>
  <property name="toolTip">
   <string>Delete single item</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>General</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QWidget" name="widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QComboBox" name="lstClasses"/>
           </item>
           <item>
            <widget class="QPushButton" name="btnAddPositive">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Add positive example</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="qt_test.qrc">
               <normaloff>:/qt_test/GeneratedFiles/plus.png</normaloff>:/qt_test/GeneratedFiles/plus.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnAddNegative">
             <property name="toolTip">
              <string>Add negative example</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="qt_test.qrc">
               <normaloff>:/qt_test/GeneratedFiles/minus.png</normaloff>:/qt_test/GeneratedFiles/minus.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="MouseLabel" name="lblInput">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QScrollArea" name="scrollAreaInput">
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>547</width>
             <height>303</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QTreeWidget" name="treeResults">
          <property name="uniformRowHeights">
           <bool>true</bool>
          </property>
          <property name="headerHidden">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="btnRefresh">
             <property name="toolTip">
              <string>Reload all</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="qt_test.qrc">
               <normaloff>:/qt_test/GeneratedFiles/refresh.png</normaloff>:/qt_test/GeneratedFiles/refresh.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnRemove">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="qt_test.qrc">
               <normaloff>:/qt_test/GeneratedFiles/delete.png</normaloff>:/qt_test/GeneratedFiles/delete.png</iconset>
             </property>
             <property name="shortcut">
              <string>Ctrl+D</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnDropAllResults">
             <property name="toolTip">
              <string>Delete all results</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="qt_test.qrc">
               <normaloff>:/qt_test/GeneratedFiles/bin.png</normaloff>:/qt_test/GeneratedFiles/bin.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Information</string>
       </attribute>
       <widget class="QPlainTextEdit" name="plainTextEdit_Console">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditSquire">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>30</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>11</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditThreshold">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>70</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>50</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>30</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Squire</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>70</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Threshold</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>605</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionDetect"/>
    <addaction name="actionBatch_detect"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_positive"/>
    <addaction name="actionAdd_negative"/>
    <addaction name="actionRemove"/>
    <addaction name="actionDrop_All_results"/>
   </widget>
   <widget class="QMenu" name="menuSetup">
    <property name="title">
     <string>Setup</string>
    </property>
    <addaction name="actionLoad_config_file"/>
    <addaction name="actionTrain"/>
    <addaction name="actionLoad_test_images"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoom_in_2"/>
    <addaction name="actionZoom_out_2"/>
    <addaction name="actionRefresh"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuSetup"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="qt_test.qrc">
     <normaloff>:/qt_test/GeneratedFiles/open.png</normaloff>:/qt_test/GeneratedFiles/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionTrain">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Train</string>
   </property>
  </action>
  <action name="actionLoad_config_file">
   <property name="text">
    <string>Load config file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionDetect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Detect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionBatch_detect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Batch detect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionZoom_in_2">
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoom_out_2">
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionLoad_test_images">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load test images</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="qt_test.qrc">
     <normaloff>:/qt_test/GeneratedFiles/refresh.png</normaloff>:/qt_test/GeneratedFiles/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionAdd_positive">
   <property name="icon">
    <iconset resource="qt_test.qrc">
     <normaloff>:/qt_test/GeneratedFiles/plus.png</normaloff>:/qt_test/GeneratedFiles/plus.png</iconset>
   </property>
   <property name="text">
    <string>Add positive</string>
   </property>
  </action>
  <action name="actionAdd_negative">
   <property name="icon">
    <iconset resource="qt_test.qrc">
     <normaloff>:/qt_test/GeneratedFiles/minus.png</normaloff>:/qt_test/GeneratedFiles/minus.png</iconset>
   </property>
   <property name="text">
    <string>Add negative</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="qt_test.qrc">
     <normaloff>:/qt_test/GeneratedFiles/delete.png</normaloff>:/qt_test/GeneratedFiles/delete.png</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionDrop_All_results">
   <property name="icon">
    <iconset resource="qt_test.qrc">
     <normaloff>:/qt_test/GeneratedFiles/bin.png</normaloff>:/qt_test/GeneratedFiles/bin.png</iconset>
   </property>
   <property name="text">
    <string>Drop All results</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MouseLabel</class>
   <extends>QLabel</extends>
   <header location="global">mouselabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qt_test.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionAdd_positive</sender>
   <signal>triggered()</signal>
   <receiver>btnAddPositive</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAdd_negative</sender>
   <signal>triggered()</signal>
   <receiver>btnAddNegative</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDrop_All_results</sender>
   <signal>triggered()</signal>
   <receiver>btnDropAllResults</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>550</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRemove</sender>
   <signal>triggered()</signal>
   <receiver>btnRemove</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>497</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRefresh</sender>
   <signal>triggered()</signal>
   <receiver>btnRefresh</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
